packages.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "onnxruntime_nim",
```

Comment:
The name needs to match your .nimble file's name :)

Reply:
Isn't using "nim" in package names disallowed anyway?

Reply:
nope